### ISSUE TITLE
FR file input

### ISSUE THREAD

**Issue by rejifald** [nvms/vue-atlas#47]
File picker with previews will be great. For example https://lusaxweb.github.io/vuesax/components/upload.html#default

**Comment by rejifald** [comment]
Btw, will be also great to merge all inputs in one input with type attribute. 
Just as native input works. 
Wdyt ?

**Comment by nvms** [comment]
good idea. if (when) this is implemented, it will be as a `<va-input type="file" ... />`

**Comment by rejifald** [comment]
Do u work with this ?
If no, let's discuss about component feature and interface, and i am ready to help.

**Comment by nvms** [comment]
I haven't had time to work on this yet. still exploring ideas. I'm not sure how it would even look quite yet

**Comment by rejifald** [comment]
It can be button with appropriate icon, and filename when file is picked

**Comment by rejifald** [comment]
But in other hand, there should accept drag and drop, and file preview. Helpful for images, but it may use appropriate icon for common file types. 
Documents, videos, audio, etc.

**Comment by rejifald** [comment]
Have u design system ? Which u use.

**Comment by rejifald** [comment]
<img width="420" alt="screen shot 2019-02-20 at 11 44 37 pm" src="https://user-images.githubusercontent.com/42000283/53126797-a2c26d80-3569-11e9-8357-653562056afa.png">
<img width="413" alt="screen shot 2019-02-20 at 11 44 22 pm" src="https://user-images.githubusercontent.com/42000283/53126798-a2c26d80-3569-11e9-98b2-cb6c6790639b.png">
Few examples of such component that i created for a long time ago.

**Comment by nvms** [comment]
looks nice, if you'd like to work on this then please go ahead. if you **do** decide to work on this, please try and make it appear visually similar to the current button/input styles

some thoughts:
- automatic preview of images should be optional
- drag and drop target should be its own component?

**Comment by rejifald** [comment]
Let's postpone drag and drop for now.
I completely agree that it should looks like current design.
Do u have design system or something like that ? If no, i just will try to make it similar with current design.

**Comment by nvms** [comment]
I do not have a design system, so you'll have to use your best judgement

**Comment by rejifald** [comment]
Ok, will do

**Comment by rejifald** [comment]
I am working on it. I've implemented component almost fully. I'm just working under it's style.
And i'm gonna to use ToggleIcon and Buttons with prefix/postfix on this file picker.
